src/english-word-club.ts

```diff
@@ -6,7 +6,7 @@ import * as fs from 'fs';
 import path from 'path';
 const mit10k_en_us_path = path.resolve(__dirname, './data/mit10k-en-US.txt');
 
-const mit10k_en_us = fs.readFileSync(mit10k_en_us_path, 'utf8').split('\n');
+const mit10k_en_us = fs.readFileSync(mit10k_en_us_path, 'utf8').split('\n').map(line => line.trim());
```

Comment:
On `windows` machine `fs.readFileSync(mit10k_en_us_path, 'utf8').split('\n')` add space`\r`, like this. `'a\r' --> 'a'` so to trim that space, it is added.

cc- @xinbenlv